jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/saved/SavedFiltersPresenter.java

```diff
@@ -105,4 +105,11 @@ public void onRestoreDefaultFilters() {
                                                                                  Commands.DO_NOTHING);
         yesNoCancelPopup.show();
     }
+
+    public void onSaveDefaultActiveFilter(@Observes final SavedFilterAsDefaultActiveEvent event) {
+        filterSettingsManager.saveDefaultActiveFilter(event.getSavedFilter().getKey(), () -> {
+            view.removeAllSavedFilters();
```

Comment:
reloading all filters seems a bit unnecessary, should just update the icon maybe?

Reply:
@cristianonicolai I added here a new call updateSavedFiltersDefault to updated the default filter avoiding reload all filters again